api-specs/user.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package userapi;
+option go_package = "/userapi";
+
+service userService {
+	rpc getProfile (getProfileRequest) returns (Profile);
+}
+
+message getProfileRequest {
+	int64 userId = 1;
+}
+
+message Profile {
```

Comment:
Should we make use of `ProfileRequest` and `ProfileResponse` as a convention?

Reply:
okay!